docs/designs/data_transform.md

```diff
@@ -168,15 +168,18 @@ After normalizing the table schema, we can do data analysis and transformation o
 
 We can extend the SQLFlow syntax and enrich the `COLUMN` expression. We propose to add some built-in functions to describe the transform process. We will implement common used functions at the first stage.  
 
-| Name          |    Transformation                                     | Statitical Parameter |
-|:-------------:|:-----------------------------------------------------:|:--------------------:|
-| NORMALIZE     | x - x_min / (x_max - x_min)                           |     x_min, x_max     |
-| STANDARDIZE   | x - x_mean / x_stddev                                 |    x_mean, x_stddev  |
-| LOG_ROUND     | tf.round(tf.log(x))                                   |          N/A         |
-| BUCKETIZE     | tf.feature_column.bucketized_column                   |    bucket_boundary   |
-| HASH          | tf.feature_column.categorical_column_with_hash_bucket |   hash_bucket_size   |
-| CROSS         | tf.feature_column.crossed_column                      |          N/A         |
-| EMBEDDING     | tf.feature_column.embedding_column                    |          N/A         |
+| Name             |    Transformation                                     | Statitical Parameter | Input Type | Output Type |
+|:----------------:|:-----------------------------------------------------:|:--------------------:|:-------------------:|:---------------------:|
+| NORMALIZE(x)     | Scale the inputs to the range [0, 1]. `out = x - x_min / (x_max - x_min)` |     x_min, x_max     | number | float64 |
```

Comment:
What does `number` mean? Including all int, float, etc?

Reply:
Yes, int, float and double